Proxy/proxy.py

```diff
@@ -262,7 +262,7 @@ def proxy():
 
             if limit is None:
                 # Optional
-                limit = len(queDict[port])
+                limit = len(getDicts[station][port])
```

Comment:
`queDict` doesn't exist. It was a guess on my part that what was meant was `getDicts[station]`. Having these presubmits in place should help avoid us introducing code like this that we can't explain in the future.